src/api.js

```diff
@@ -46,4 +46,15 @@ export const updateArticleVotes = (article_id) => {
     })
     .then((res) => {
     })
+};
+
+export const fetchArticleComments = (article_id) => {
+  return axios
+    .get(`https://news-api-08-22.herokuapp.com/api/articles/${article_id}/comments`)
+    .then((res) => {
+      return res.data;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
```

Comment:
remember by catching here you're stopping yourself from catching and handling in the React component!

What would you want to show your users if the fetching of comments failed?